### PR TITLE
Adding idea project files to .gitignore

### PR DESCRIPTION
**Pull request by ethteck**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -82,7 +82,8 @@ hs_err_pid*.log
 /RecentActivity/release/
 /CentralRepository/release/
 
-/.idea/
+.idea/
+*.iml
```
Comment (rcordovano) [inline .gitignore]: What are these files?

**Comment by ethteck** [comment]
These are Intellij project files

File: .gitignore (hunk 1 above)
Comment (rcordovano) [reply .gitignore]: What is the context for the presence of Intellij project files in Autopsy?

**Comment by ethteck** [comment]
I've been trying to get Intellij development working and have been
moderately successful.

File: .gitignore (hunk 1 above)
Comment (rcordovano) [reply .gitignore]: It seems like adding these files to .gitignore is premature.

**Comment by ethteck** [comment]
Alright. Well one was already added recently in a different PR of mine; I'm
just fixing it to include not just the top level directory. The other is an
extension that nothing else uses as far as I know. I can just add them to
my global .gitignore though if you're against it.

**Pull request merged**